### PR TITLE
Added Conference Times and updated conference description 

### LINKED ISSUE #50: Dates for in-person & online events
# Americas online session
2024-05-08 Wednesday 11:00 - 15:00 US Eastern Standard Time  / 15:00 - 19:00 UTC / 17:00 - 21:00 Central European Summer Time

# Asia Pacific & Americas online session
2024-05-09 Thursday 19:00 - 23:00 US Eastern Standard Time 
2024-05-10 Friday, May 10th 00:00 - 04:00 UTC / 09:00 - 13:00 Japanese Standard Time  / 10:00 - 14:00 Australian Eastern Standard Time

# In Amsterdam
(CEST = [Central European Summer Time](https://en.wikipedia.org/wiki/Central_European_Summer_Time))

2024-05-14 Tuesday 13:00-17:30 CEST Novice Training (in-person only)

2024-05-15 Wednesday 08:30-12:30 CEST In-person conference day 1, morning session (hybrid & auditorium) (06:30 - 10:30 UTC)
2024-05-15 Wednesday 12:30-13:30 CEST Lunch
2024-05-15 Wednesday 13:30-17:30 CEST In-person conference day 1, afternoon session (hybrid & auditorium) (11:30 - 16:30 UTC)

2024-05-16 Thursday 08:00-12:00 CEST In-person conference day 2, morning session (hybrid & auditorium) (06:00 - 10:00 UTC)
2024-05-16 Thursday 12:00-13:00 CEST Lunch
2024-05-16 Thursday 13:00-17:00 CEST Break out chats (small groups, in-person only)

2025-05-17 Friday 09:00-17:30 CEST Co-working (hackathon, in-person & maybe hybrid) (07:00 - 15:30 UTC)

### PR DESCRIPTION
**Pull request by alexiswl**
Resolves #50

### CODE REVIEW

File: cwlcon2024-site/index.html [hunk 1]
```diff
@@ -130,7 +135,135 @@ <h2><a href="#registration">Registration</a></h2>
                     <div id="program" class="column">
                         <h2><a href="#program">Program</a></h2>
 
-                        <p>Please check again soon!</p>
+                        <br>
+
+                        <h3><a href="#onlinesession1">Americas Online Session</a></h3>
+
+                        <p>
+                            A four-hour online conference session for America / EMEA timezones.<br>
```
Comment (mr-c) [inline cwlcon2024-site/index.html:143]:
```suggestion
                            A four-hour online conference session for those in the Americas, Europe, Middle East, and Africa.<br>
```

File: cwlcon2024-site/index.html [hunk 2]
```diff
@@ -130,7 +135,135 @@ <h2><a href="#registration">Registration</a></h2>
...
+                            </tr>
+                            </tbody>
+                        </table>
+
+                        <br><br>
+
+                        <h3><a href="#onlinesession2">APAC & Americas Online Session</a></h3>
+
+                        <p>
+                            A four-hour online conference session for Asia-Pacific / America timezones.<br>
```
Comment (mr-c) [inline cwlcon2024-site/index.html:185]: similiar to above, lets be consistent with "America**s**"

File: cwlcon2024-site/index.html [hunk 3]
```diff
@@ -130,7 +135,135 @@ <h2><a href="#registration">Registration</a></h2>
...
+                                <th class="tg-0pky">Date</th>
+                                <th class="tg-0pky">Time</th>
+                                <th class="tg-0pky">Event Description</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            <tr>
+                                <td class="tg-0pky">Tue 14 May</td>
+                                <td class="tg-0pky">13:00 - 17:30</td>
+                                <td class="tg-0pky">Novice Training</td>
```
Comment (mr-c) [inline cwlcon2024-site/index.html:237]:
```suggestion
                                <td class="tg-0pky">Novice Training (Separate signup, limited room)</td>
```

File: cwlcon2024-site/index.html [hunk 4]
```diff
@@ -130,7 +135,135 @@ <h2><a href="#registration">Registration</a></h2>
...
+                            </tr>
+                            <tr>
+                                <td class="tg-0pky">Wed 15 May</td>
+                                <td class="tg-0pky">08:00 - 18:00</td>
+                                <td class="tg-0pky">In-person conference day 1 (auditorium)</td>
```
Comment (mr-c) [inline cwlcon2024-site/index.html:242]:
```suggestion
                                <td class="tg-0pky">In-person conference day 1</td>
```

File: cwlcon2024-site/index.html [hunk 5]
```diff
@@ -31,7 +31,7 @@ <h1 role="doc-subtitle">
         </div>
         <!-- This banner is duplicated below for larger viewports. Please update it there as well. -->
         <div class="banner-text">
-            <span>Week of May 13th, 2024</span>
+            <span>14th - 17th May 2024</span>
```
Comment (mr-c) [inline cwlcon2024-site/index.html:34]: I think we need more words here


The in-person conference proper is only May 15&16th, the optional novice training session is on the 14th (which will have a separate signup), and optional hackathon day is May 17th. Not to mention the online-only sessions the week before. Hmm..

**Review by mr-c — CHANGES_REQUESTED** [review]
Thank you @alexiswl for starting this!

File: cwlcon2024-site/index.html (hunk 5 above)
Comment (alexiswl) [reply cwlcon2024-site/index.html:34]: Yes I played around with this and nothing seemed to fit nicely.  We could do just 8th - 17th May 2024 but don't want to confuse people before they look at the program

**Review by kinow — APPROVED** [review]
Thank you!

File: cwlcon2024-site/index.html (hunk 5 above)
Comment (alexiswl) [reply cwlcon2024-site/index.html]: Have moved to 8th to 17th

**Pull request merged**